index.html

```diff
@@ -135,7 +135,9 @@
             <div id="plotOutput"></div>
           </div>
           <div id="statsTab" class="tab-pane fade">
-            <div id="statsOutput"></div>
+           <div id="statsPlot1"></div>
```

Comment:
1. Why do we need two stats plot divs?
2. Can we give them more informative names than `1` and `2`?

Reply:
I didn't know if vegaEmbed could handle an array of specs, plotting two plots within a single div? I'll look into it